lib/smci_report.rb

```diff
@@ -109,13 +109,13 @@ def run
 
           if author # we found the author in our database, now get their publications
             contributions = Contribution.select('*')
-            contributions = contributions.where("author_id = ? and status in ('approved')", author.id)
+            contributions = contributions.where("author_id = ? and status in ('new','approved')", author.id)
```

Comment:
can you do `contributions.where(author: author).where(status: %w[new approved])`?

Reply:
Yup, good call